### PR TITLE
describe server extension suite

### PR DESCRIPTION
**Pull request by mcfnord**
This PR adds coverage of server extension tools [described in this discussion](https://github.com/orgs/jamulussoftware/discussions/3085). The tools all work together, so I cover them together.

### CODE REVIEW

**Review by ann0see — COMMENTED** [review]
Looks good. I think we could also add Koord and a lot of other stuff.

**Comment by mcfnord** [comment]
> I think we could also add Koord

What if I wrote: "Koord.Live pairs Jamulus with real-time video of participants in one application. The website offers private servers and additional playback and recording features."

**Comment by ann0see** [comment]
Yes. And maybe also add that they provide iOS and Android apps on the AppStore and Play Store.

**Comment by ann0see** [comment]
@gilgongo could you please review this?

File: _posts/2022-01-25-Related-Projects.md [hunk 1]
```diff
@@ -56,3 +56,16 @@ within a Docker container. It provides a `Dockerfile` and some documentation.
...
+
+[These components](https://github.com/orgs/jamulussoftware/discussions/3085) extend the features of a Jamulus server.
+
+* [jamulus-lounge](https://github.com/dtinth/jamulus-lounge) adds a web-based interface that lets people listen to a Jamulus server. It connects to a server using a bot named **lobby**.
+* [gojam](https://github.com/dtinth/gojam) is an implementation of a Jamulus client written in Go. This tool is intended for headless streaming use-cases. Used by **jamulus-lounge**.
+* [jamviz](https://github.com/dtinth/jamviz) shows active connections of a Jamulus server that is running **jamulus-lounge**. Used to show activity to a livestreaming audience.
+
+## Koord
+
+Koord pairs Jamulus with real-time video of participants in one application, available for desktop and mobile devices. The [Koord.live website](https://koord.live/) offers private servers and additional playback and recording features.
```
Comment (ann0see) [inline _posts/2022-01-25-Related-Projects.md]: Maybe better say something like Koord provides a Jamulus compatible app. 

Their app feels quite different to the official one. 

Another thing: To provide equal opportunities, https://melomax.live/ should be mentioned too as Jamulus Server Hoster comparable to Koord.

**Review by ann0see — COMMENTED** [review]
Thanks!

File: _posts/2022-01-25-Related-Projects.md (hunk 1 above)
Comment (mcfnord) [inline _posts/2022-01-25-Related-Projects.md]:
```suggestion
Koord adds real-time video of participants to Jamulus technology in a new application, available for desktop and mobile devices. The [Koord.live website](https://koord.live/) offers private servers and additional playback and recording features.

## Melomax

The [Melomax website](https://melomax.live) offers private servers groups can use with Jamulus client software.

```

Comment (ann0see) [reply _posts/2022-01-25-Related-Projects.md]: Thanks

**Review by ann0see — APPROVED** [review]
Actually, as it's a KB entry, we can just merge it.

**Pull request merged**